docs/integrations/datadog-rum/installation.md

```diff
@@ -28,6 +28,51 @@ app:
 
 If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/packages/app/public/index.html#L69) to include the Datadog RUM `init()` section manually.
 
+Please note that the env value MUST be specified at build time
+
+:::note
+In case after a proper configuration, the events still are not being captured: Copy and paste this section in to your `packages/app/public/index.html` under the `<head>` tag.
+
+```html
+<% if (config.has('app.datadogRum')) { %>
+<script>
+  (function (h, o, u, n, d) {
+    h = h[d] = h[d] || {
+      q: [],
+      onReady: function (c) {
+        h.q.push(c);
+      },
+    };
+    d = o.createElement(u);
+    d.async = 1;
+    d.src = n;
+    n = o.getElementsByTagName(u)[0];
+    n.parentNode.insertBefore(d, n);
+  })(
+    window,
+    document,
+    'script',
+    'https://www.datadoghq-browser-agent.com/datadog-rum-v3.js',
+    'DD_RUM',
+  );
+  DD_RUM.onReady(function () {
+    DD_RUM.init({
+      clientToken: '<%= config.getString("app.datadogRum.clientToken") %>',
+      applicationId: '<%= config.getString("app.datadogRum.applicationId") %>',
+      site: '<%= config.getOptionalString("app.datadogRum.site") || "datadoghq.com" %>',
+      service: 'backstage',
+      env: '<%= config.getString("app.datadogRum.env") %>',
+      sampleRate:
+        '<%= config.getOptionalNumber("app.datadogRum.sessionSampleRate") || 100 %>',
```

Comment:
This should either be:

`<%= config.getOptionalNumber("app.datadogRum.sessionSampleRate") || 100 %>` (without the `'` quotes)

or:

`Number('<%= config.getOptionalNumber("app.datadogRum.sessionSampleRate") || 100 %>')`

The same applies to `sessionReplaySampleRate` below. Otherwise, the Datadog RUM lib will complain and no data will be sent.